openjdk-runtime/openjdk8/Dockerfile

```diff
@@ -16,9 +16,9 @@ FROM gcr.io/google_appengine/base
 ENV DEBIAN_FRONTEND noninteractive
 
 # Create env vars to identify image
-ENV GAE_IMAGE_NAME ${project.artifactId}
-ENV GAE_IMAGE_LABEL ${docker.label.prefix}8-jre
-ENV GAE_IMAGE_VERSION ${project.version}
+# ENV GAE_IMAGE_NAME ${project.artifactId}
```

Comment:
Having a mechanism to identify the version of an image is really valuable.  Once you start trying to support problems with deployed images, it can be very difficult to work out exactly what base image was used.      It would be good to establish a standard for this over all google images